sched/generators.go

```diff
@@ -84,7 +84,7 @@ func GenTask() TaskDefinition {
 func GenRandomTask(rng *rand.Rand) TaskDefinition {
 	snapshotId := fmt.Sprintf("snapShotId:%s", testhelpers.GenRandomAlphaNumericString(rng))
 	taskId := fmt.Sprintf("taskId:%s", testhelpers.GenRandomAlphaNumericString(rng))
-	numArgs := rng.Intn(5)
+	numArgs := rng.Intn(5) + 1 // must have at least 1 arg
```

Comment:
I might be wrong about this particular instance, but generally we want to test -all- combinations of inputs, including invalid ones.